stroyprombeton/tests/tests_views.py

```diff
@@ -267,7 +267,7 @@ def test_fetch_positions_searching(self):
 
     def test_products_are_from_category(self):
```

Comment:
Let's create todo to fix these buggy tests. They should rely upon hard-coded data

Reply:
shouldn't *rely